src/viadot/utils.py

```diff
@@ -1092,3 +1092,47 @@ def df_clean_column(
                 inplace=True,
             )
     return df
+
+
+def handle_empty_df(
+    if_df_empty: Literal["warn", "skip", "fail"],
+    message: str
```

Comment:
Add a default value to the message.
suggestion: `message: str = "The query produced no data."`

The suggestion is the same as the version used in the `Base._handle_if_empty()`